src/workerd/io/hibernation-manager.h

```diff
@@ -103,18 +116,41 @@ class HibernationManagerImpl final : public Worker::Actor::HibernationManager {
       }
     }
 
+    // TODO(now): We can probably return an ArrayPtr<StringPtr> instead, and then when we
```

Comment:
I'm not positive about this, but we need an IoOwn to get the tags anyway, right? Could we just make the tags a refcounted type and avoid passing it back and forth for the most part? We should only be touching it when we're on the io thread anyway.

Reply:
By IoOwn you mean our `farNative` websocket, yeah? In other words, we would only call this when JS is running.

Is the idea here to have the tags stored in an Array that's allocated with `kj::refcounted`, and then just take a strong ref whenever we need the tags in JS? I suppose that would work but we would still need to give the `api::WebSocket` a strong ref when we dispatch a close.

Reply:
I think the idea was to keep a strong ref in both places. I guess we'd make a new strong reference when we instantiate the `api::WebSocket`. Basically, we stop worrying about move semantics and just say that it exists.